### PR TITLE
Research: How to fetch the list of articles from Wikidata API #13

### PR DESCRIPTION
**Pull request by GonzaFP**
Instead of manually copying and pasting article names, I have written a guide on how to fetch and extract articles from Wikidata API

### CODE REVIEW

File: data-gathering/__pycache__/bot.cpython-311.pyc
Comment (wikicurricula-uy) [inline data-gathering/__pycache__/bot.cpython-311.pyc:1]: Please remove this

File: README.md
Comment (wikicurricula-uy) [inline README.md:1]: Can you help me understand which parts of the readme you've changed? Too many lines are highlighted and it's confusing

File: visualization/assets/script/dv1.js
Comment (wikicurricula-uy) [inline visualization/assets/script/dv1.js:1]: Do we have any actual changes in dv1.js? Or is it all whitespaces changes and such?

File: visualization/assets/script/utilities.js
Comment (wikicurricula-uy) [inline visualization/assets/script/utilities.js:1]: Do we have any actual changes in utilities.js? Or is it all whitespaces changes and such?

**Review by wikicurricula-uy — CHANGES_REQUESTED** [review]
I'm sorry, @GonzaFP , do you think you can clean up this PR a bit? I see that 12 files changed in the comparison but that's not correct, right? Which files are you changing in order to achieve this?

**Comment by GonzaFP** [comment]
@wikicurricula-uy  thanks for the feedback. I only changed the README but let me clean up the PR.

File: visualization/assets/script/utilities.js
Comment (GonzaFP) [reply visualization/assets/script/utilities.js:1]: Hello @wikicurricula-uy , It is whitespaces.

File: README.md
Comment (GonzaFP) [reply README.md:1]: @wikicurricula-uy  this is what I added https://github.com/GonzaFP/wikicurricula-boilerplate/blob/new-branch/README.md#how-to-fetch-the-list-of-articles-from-wikidata-api

File: data-gathering/__pycache__/bot.cpython-311.pyc
Comment (GonzaFP) [reply data-gathering/__pycache__/bot.cpython-311.pyc:1]: @wikicurricula-uy I have removed it. 
Thanks

File: README.md
Comment (wikicurricula-uy) [reply README.md:1]: Ok, I think this is already covered by this pull request which was already merged https://github.com/wikicurricula-uy/wikicurricula-boilerplate/pull/57
Let me know if I'm overlooking anything.
I'll close this PR, thanks for your work!

**Pull request closed without merging**